src/libertem/api.py

```diff
@@ -510,6 +516,10 @@ def run_udf(self,
         .. versionchanged:: 0.6.0
             Added the :code:`corrections` and :code:`backends` parameter
 
+        .. versionchanged:: 0.7.0
+            Added the :code:`plots`, :code:`sync`, and :code:`iterate` parameters,
+            and the ability to run multiple UDFs on the same data at the same time.
+
```

Comment:
The documentation of the `udf` parameter should include info that now iterables of UDFs are also allowed.